.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,404 @@
+name: Unit and Integration Tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Not used, but I'll remove all "auto" triggers, because this workflow will be triggered from other workflow.

Will be using `on: [workflow_dispatch, workflow_call]` instead